src/js/letters/actions/letters.js

```diff
@@ -215,10 +227,18 @@ export function saveAddressFailure(address) {
 }
 
 export function saveAddress(address) {
+  const transformedAddress = Object.assign({}, address);
+  if (transformedAddress.type === addressTypes.military) {
```

Comment:
I assume the code to actually set the `type` before saving will be somewhere else?

Reply:
Yup! In another PR.